### PR TITLE
Fixed broken links - ESF, Tabs, Bottom Nav & Navbar - vNext

### LINKED ISSUE #2743: Broken links in the ESF, Tabs, Bottom Nav & Navbar topics
### Excel Style Filtering

Templates:
- IgxExcelStyleColumnOperationsTemplateDirective
- IgxExcelStyleFilterOperationsTemplateDirective

External Excel Style Filtering/Usage:
- column (x2)

Styling:
- Palettes

### Tabs

Usage:
- igx-tab-header
- igx-tab-content

Customizing Tabs:
- igxTabHeaderIcon
- igx-tab-header (x2)

### Bottom Navigation:

API References:
- IgxTabComponent

### Navbar

Using color palettes:
- Palettes

### PR DESCRIPTION
**Pull request by igdmdimitrov**
Closes #2743 

### Checklist:
 
 - [ ] check topic's TOC/menu and paragraph headings
 - [ ] link to other topics using `../relative/path.md`
 - [ ] at the References section at the end of the topic add links to topics, samples, etc
 - [ ] reference API documentation instead of adding a section with API

<br />

 - [ ] use valid component names - [Data] Grid, `IgxSelectComponent`, `<igx-combo>`
 - [ ] use spell checker tool (VS Code, Grammarly, Microsoft Editor)
 - [ ] add inline `code blocks` for the names of classes / tags / properties
 - [ ] add language descriptor for the ```code blocks```
 - [ ] check broken links (use browser add-on)
 - [ ] check if sample is working and fully visible in the topic
 - [ ] check if sample is working and fully visible in the StackBlitz
 - [ ] check if code blocks match the code in StackBlitz demo

<br />

 - [ ] follow SEO guidelines to fill topic's metadata ([SEO guidelines](https://infragisticsinc297.sharepoint.com/:w:/g/Groups/marketing/EWz9InT4FDlErHCumxsKGY4Bd8H03yhRWxDFk47luRz-_Q?e=S5wWcx))

<br />

 - [ ] do not resolve requested changes (leave that to the reviewer)
 - [ ] add `pending-localization` label when the review of the PR is done
 - [ ] add a member from the localization team to translate it

### CODE REVIEW

**Review by gedinakova — APPROVED** [review]

File: en/components/grids_templates/excel-style-filtering.md [hunk 1]
```diff
@@ -214,8 +214,8 @@ In the sample below 'Artist' column have all four features enabled, 'Debut' have
 ## Templates
 
 If you want to further customize the Excel style filter menu without disabling the column features you could use custom templates. The Excel Style filter menu provides two directives for templating:
-- [`IgxExcelStyleColumnOperationsTemplateDirective`]({environment:angularApiUrl}/classes/igxexcelstylecolumnoperationsdirective.html) - re-templates the area with all column operations like sorting, moving, pinning, etc.
-- [`IgxExcelStyleFilterOperationsTemplateDirective`]({environment:angularApiUrl}/classes/igxexcelstylefilteroperationsdirective.html) - re-templates the area with all filter specific operations.
+- [`IgxExcelStyleColumnOperationsTemplateDirective`]({environment:angularApiUrl}/classes/igxexcelstylecolumnoperationstemplatedirective.html) - re-templates the area with all column operations like sorting, moving, pinning, etc.
```
Comment (gedinakova) [inline en/components/grids_templates/excel-style-filtering.md:217]: I'd rather remove the links to these two directives. They have no API members so the page we link to doesn't really bring any value.

File: en/components/tabs.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ import { IgxTabsModule } from 'igniteui-angular';
 export class AppModule {}
 ```
 
-We set the tabs header by providing content to [`igx-tab-header`]({environment:angularApiUrl}/classes/igxtabheader.html}). To set the tab's name we simply add a span with [`igxTabHeaderLabel`]({environment:angularApiUrl}/classes/igxtabheaderlabeldirective.html) directive. Any content that will appear as a tab item's content should be added between [`igx-tab-content`]({environment:angularApiUrl}/classes/igxtabcontent.html}) tags.
+We set the tabs header by providing content to `igx-tab-header`. To set the tab's name we simply add a span with [`igxTabHeaderLabel`]({environment:angularApiUrl}/classes/igxtabheaderlabeldirective.html) directive. Any content that will appear as a tab item's content should be added between `igx-tab-content` tags.
```
Comment (gedinakova) [inline en/components/tabs.md:41]: Same as above, let's remove the link to igxTabHeaderLabel directive as well. We just need to know its name.

File: en/components/tabs.md [hunk 3]
```diff
@@ -121,7 +121,7 @@ Let's modify the tabs and make them more appealing by including icons using the
 > [!NOTE]
 > If you haven't used the `igx-icon` in your application so far, please make sure to import the `IgxIconModule` in the **app.module.ts** before proceeding.
 
-First, add the Material Icons in your 'styles.scss' file in the main application folder. Next, add `igx-icon` with [`igxTabHeaderIcon`]({environment:angularApiUrl}/classes/igxtabheadericondirective.html}) directive set, as a child of [`igx-tab-header`]({environment:angularApiUrl}/classes/igxtabheader.html}) .
+First, add the Material Icons in your 'styles.scss' file in the main application folder. Next, add `igx-icon` with [`igxTabHeaderIcon`]({environment:angularApiUrl}/classes/igxtabheadericondirective.html) directive set, as a child of `igx-tab-header`.
```
Comment (gedinakova) [inline en/components/tabs.md:124]: Same as above.

**Review by gedinakova — CHANGES_REQUESTED** [review]

**Review by gedinakova — CHANGES_REQUESTED** [review]
There is one more broken link - "If setting the labels and icons is not enough, you can also provide your own custom content directly between **igx-tab-header** tags."

**Comment by igdmdimitrov** [comment]
> There is one more broken link - "If setting the labels and icons is not enough, you can also provide your own custom content directly between **igx-tab-header** tags."

fixed

**Review by gedinakova — APPROVED** [review]

**Pull request merged**